examples/fluent-bit/log-driver-buffer-limit/README.md

```diff
@@ -0,0 +1,28 @@
+### FireLens Example: Specifying buffer limit size with 'Fluentd' log driver
+
+`log-driver-buffer-limit` option is now supported on ECS EC2 and ECS Fargate with PV1.4. This setting helps FireLens to configure the Fluentd Docker Log Driver field [fluentd-buffer-limit](https://docs.docker.com/config/containers/logging/fluentd/#fluentd-buffer-limit). It will specify the limit size for number of events buffered on the memory to avoid log loss issue when buffer is full. 
```

Comment:
More details need to be added from your design doc. This is a fairly complicated feature and I think even showing the diagram would be useful in our docs, in which we can give lots of detail. You can explain how the buffer is used in the log driver in the FireLens architecture so that users know exactly what this does. I am worried a naive user might think that this feature modifies some buffer inside fluent bit process. Or that others might not know how the log driver is used in FireLens.

And then go into slightly more detail on why you would want to use this- the buffer can get full under high load and drop logs.